[MTM-63637] change log for stricter validation of email address (#4652)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-2026-128-0-strict-validation-of-user-email-address.md

---
date: '2026-04-20'
title: Stricter validation of user email addresses
product_area: Platform services
change_type:
  - value: change-3BQrQ6adS
    label: API change
component:
  - value: component-JlFdtOPva
    label: REST API
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-63637
version: 2026.128.0
---

> We are introducing stricter validation for user email addresses.

Reviewer: ```suggestion
We are introducing a stricter validation for the user email addresses.
```

Author: This change log is a copy of the change log in c8y-docs-ops (where it should not be). It has been modified according to proposals by Jane Porter (native speaker), and I would like to keep this result.

Reviewer 2: Agreed. Let´s stick with Jane's wording here. It does not always follow our guidelines but here it does :-).

> The domain part after the '@' symbol may now only include

Reviewer: ```suggestion
The part of the domain name following the ‘@’ symbol may now only contain
```

Author: same as above:
This change log is a copy of the change log in c8y-docs-ops (where it should not be). It has been modified according to proposals by Jane Porter (native speaker), and I would like to keep this result.

Reviewer 2: Here I partly agree with Jochen, will change it accordingly.
